owncloudApp/src/main/res/layout/numberkeyboard_layout.xml

```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You forgot to add the copyright to the file! Don't worry, ask us how to configure it and we will help you 😉